src/event/components/DayPicker.vue

```diff
@@ -1,76 +1,86 @@
 <template>
-  <table>
-    <tr>
-      <th v-for="(day, i) in days" :key="i" class="body2">{{ day[0] }}</th>
-    </tr>
-    <tr>
-      <td
+  <div class="day-picker">
+    <!-- Day names ordered via data() -->
+    <ul class="days button">
+      <li
         v-for="(day, i) in days"
         :key="i"
+        class="day-name button"
         :class="{ selected: day[1] }"
+      >
+        {{ day[0] }}
+      </li>
+    </ul>
+
+    <!-- Selectable day blocks -->
+    <ul class="days button">
+      <li
+        v-for="(day, i) in days"
+        :key="i"
+        :class="{ selected: day[1] }"
+        class="day-block"
         @click="day[1] = !day[1]"
-      ></td>
-    </tr>
-  </table>
+      ></li>
+    </ul>
+  </div>
 </template>
 
-<script>
-//TODO: Make buttons for day selection
-//FIXME: Flexbox instead of tables
-//FIXME: Styling
-//FIXME: Relative units
+<script lang="ts">
 import { defineComponent } from "vue";
 export default defineComponent({
   name: "DayPicker",
 
   data() {
     return {
       // Each day of the week has name and selected state
+      // Ordered as listed
       days: [
+        ["Sun", false],
```

Comment:
Small nit, but it might be better for days to be an object rather than a tuple. Something like:

```
{
  day: "Sun",
  isSelected: false
}
```

Currently, it's not super obvious to me what the `false` corresponds to here.

Reply:
true just added this